obonet/read.py

```diff
@@ -36,7 +36,7 @@ def read_obo(path_or_file):
         if is_obsolete:
             continue
         term_id = term.pop('id')
-        graph.add_node(term_id, attr_dict=term)
+        graph.add_node(term_id, **term)
```

Comment:
So this is the only compatibility change required? That was less painful than expected 😸